pkg/codegen/internal/test/testdata/simple-plain-schema/docs/provider.md

```diff
@@ -0,0 +1,265 @@
+    </dt>
+    <dd>
+      Bag of options to control resource&#39;s behavior.
+    </dd></dl>
+
+{{% /choosable %}}
+
+## Provider Resource Properties {#properties}
+
+To learn more about resource properties and how to use them, see [Inputs and Outputs]({{< relref "/docs/intro/concepts/inputs-outputs" >}}) in the Programming Model docs.
+
+### Inputs
+
+The Provider resource accepts the following [input]({{< relref "/docs/intro/concepts/inputs-outputs" >}}) properties:
+
+
+
+{{% choosable language csharp %}}
+<dl class="resources-properties"></dl>
+{{% /choosable %}}
+
+{{% choosable language go %}}
+<dl class="resources-properties"></dl>
+{{% /choosable %}}
+
+{{% choosable language nodejs %}}
+<dl class="resources-properties"></dl>
+{{% /choosable %}}
+
+{{% choosable language python %}}
+<dl class="resources-properties"></dl>
+{{% /choosable %}}
+
+
+### Outputs
+
+All [input](#inputs) properties are implicitly available as output properties. Additionally, the Provider resource produces the following output properties:
+
+
+
+{{% choosable language csharp %}}
+<dl class="resources-properties"><dt class="property-"
+            title="">
+        <span id="id_csharp">
+<a href="#id_csharp" style="color: inherit; text-decoration: inherit;">Id</a>
+</span>
+        <span class="property-indicator"></span>
+        <span class="property-type">string</span>
+    </dt>
+    <dd>{{% md %}}The provider-assigned unique ID for this managed resource.{{% /md %}}</dd></dl>
+{{% /choosable %}}
+
+{{% choosable language go %}}
+<dl class="resources-properties"><dt class="property-"
+            title="">
+        <span id="id_go">
+<a href="#id_go" style="color: inherit; text-decoration: inherit;">Id</a>
+</span>
+        <span class="property-indicator"></span>
+        <span class="property-type">string</span>
+    </dt>
+    <dd>{{% md %}}The provider-assigned unique ID for this managed resource.{{% /md %}}</dd></dl>
+{{% /choosable %}}
+
+{{% choosable language nodejs %}}
+<dl class="resources-properties"><dt class="property-"
+            title="">
+        <span id="id_nodejs">
+<a href="#id_nodejs" style="color: inherit; text-decoration: inherit;">id</a>
+</span>
+        <span class="property-indicator"></span>
+        <span class="property-type">string</span>
+    </dt>
+    <dd>{{% md %}}The provider-assigned unique ID for this managed resource.{{% /md %}}</dd></dl>
+{{% /choosable %}}
+
+{{% choosable language python %}}
+<dl class="resources-properties"><dt class="property-"
+            title="">
+        <span id="id_python">
+<a href="#id_python" style="color: inherit; text-decoration: inherit;">id</a>
+</span>
+        <span class="property-indicator"></span>
+        <span class="property-type">str</span>
+    </dt>
+    <dd>{{% md %}}The provider-assigned unique ID for this managed resource.{{% /md %}}</dd></dl>
+{{% /choosable %}}
+
+
+
```

Comment:
Minor nit: seems like a bunch of additional empty lines are being rendered. Probably just an existing template rendering bug in which case we can fix that separately.